{"name": "Проверка возвращаемого кода 404 при логине несуществующим курьером", "status": "passed", "start": 1723291821023, "stop": 1723291821408, "uuid": "e6a95a1b-8374-4adf-9f45-0f89d9eb7230", "historyId": "fa25963727f53b3f362cf20a8027a5d7", "testCaseId": "fa25963727f53b3f362cf20a8027a5d7", "fullName": "tests.test_authorization_courier.TestAuthCourier#test_login_nonexistent_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_authorization_courier"}, {"name": "subSuite", "value": "TestAuthCourier"}, {"name": "host", "value": "MacBook-Pro-Petr.local"}, {"name": "thread", "value": "20077-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_authorization_courier"}]}